### ISSUE TITLE
Initials matching a title are eaten as title

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gdsaxton/python-nameparser#9]
```
In a name like "A.B. Vajpayee", "A.B." is incorrectly classified as a title, 
because is_title() removes periods indiscriminately.
```

Original issue reported on code.google.com by `jayqhac...@gmail.com` on 7 Feb 2012 at 8:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `dere...@gmail.com` on 7 Oct 2012 at 10:50
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Issue 12 has been merged into this issue.
```

Original comment by `dere...@gmail.com` on 7 Oct 2012 at 10:51

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in v0.2.4
```

Original comment by `dere...@gmail.com` on 11 Feb 2013 at 3:45
- Changed state: **Fixed**
